### PR TITLE
bugfix(active_campaign): fixes for intermittent errors and other enhancements

### PR DESCRIPTION
**Pull request by utsabc**
## Description of the change

> List of Fixes introduced
- API call sequence changed for contact creation
- Reduced no of API calls for custom field mapping fixed to a sing call for field mapping

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: v0/destinations/active_campaign/data/ACIdentify.json [hunk 1]
```diff
@@ -1,21 +1,26 @@
...
+    "destKey": "firstName",
+    "sourceKeys": "firstName",
+    "sourceFromGenericMap": true,
+    "required": false
+  },
+  {
+    "destKey": "lastName",
+    "sourceKeys": "lastName",
+    "sourceFromGenericMap": true,
     "required": false
```
Comment (ItsSudip) [inline v0/destinations/active_campaign/data/ACIdentify.json:24]: We shouldn't remove this as during the syncContact function call fieldvalues will not be there.

Comment (utsabc) [reply v0/destinations/active_campaign/data/ACIdentify.json:24]: We are mapping it separately in transform.js L407

**Review by shrouti1507 — APPROVED** [review]
LGTM

File: v0/destinations/active_campaign/data/ACIdentify.json (hunk 1 above)
Comment (ItsSudip) [reply v0/destinations/active_campaign/data/ACIdentify.json:24]: Yeah, that is there, but before we were sending the fieldValues during the first call but now during the syncContact call fieldValues will not be there.

Comment (utsabc) [reply v0/destinations/active_campaign/data/ACIdentify.json:24]: Earlier it was wrongly mapped we were not sending actually

Comment (ItsSudip) [reply v0/destinations/active_campaign/data/ACIdentify.json:24]: Okay then.

**Review by akashrpo — APPROVED** [review]

**Review by ItsSudip — APPROVED** [review]

**Pull request merged**